zabbix_metrics_watcher.py

```diff
@@ -214,14 +234,32 @@ def run(zabbix_host, osbs_master, config, instance):
 
             logger.info("Found build %s in %s", build_name, status)
             build = Build(build_name, cmd_base)
-            if status == 'Pending':
+            if status == 'New':
+                now = datetime.datetime.now()
+                builds_in_new.setdefault(build_name, now)
+
+            elif status != 'New':
+                try:
+                    del builds_in_new[build_name]
```

Comment:
Agree with @lcarva, we should avoid exceptions here. What if the command starts while a build is in progress, and the first event it receives is that the build has finished?